pipeline_tools/pipelines/smartseq2/smartseq2.py

```diff
@@ -274,13 +274,27 @@ def create_multi_sample_ss2_inputs_tsv_from_analysis_metadata(metadata_file):
         ['rsem_ref_index', metadata['inputs']['rsem_ref_index']],
         ['stranded', metadata['inputs']['stranded']],
         ['input_ids', metadata['inputs']['input_ids']],
+        ['stranded', metadata['inputs']['stranded']]
     ]
+
+    fastq1_input_files_content = metadata['inputs']['fastq1_input_files']
+    input_ids_content = metadata['inputs']['input_ids']
+    fastq2_input_files_content = []
+
     if metadata['inputs'].get('fastq2_input_files'):
-        content.append(
-            ['fastq2_input_files', metadata['inputs'].get('fastq2_input_files')]
-        )
+        fastq2_input_files_content = metadata['inputs']['fastq2_input_files']
 
     with open('inputs.tsv', 'w') as inputs_tsv:
         writer = csv.writer(inputs_tsv, delimiter='\t')
         writer.writerow(HEADERS)
         writer.writerows(content)
+
```

Comment:
so far I have only been focused on optimus. I moved a couple of things over from gordon's pr that had to do with ss2, but I have not implemented ss2-anything in the new scripts

Reply:
this stuff that gordon added on his latest PR I don't believe matters right now...I could be wrong but this has to do with passing the sequencing input info in through a tsv rather than grabbing it from the metadata field